docs/userGuide/reusingContents.md

```diff
@@ -257,6 +267,33 @@ In other words, **`<include>` interprets the reused code relative to the origina
 
 <hr><!-- ======================================================================================================= -->
 
+#### Specifying Variables in the Include Tag
```

Comment:
1. Let's follow the same heading level as "\<include> Inside an Included File".
1. Let's use "\<include>" rather than "Include Tag" to follow the style of the rest of the documentation.

```markdown
##### Specifying Variables in an `<include>`
```